<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GAMMA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/new/prefix1/iconos/matriz.png</normaloff>:/new/prefix1/iconos/matriz.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="proyecto"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1279</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivos">
    <property name="title">
     <string>Archivos</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbri_Archivo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionNuevo_Proyecto"/>
    <addaction name="actionNuevo_Archivo"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_como"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionBarra_de_Herramientas"/>
    <addaction name="actionArbol"/>
    <addaction name="actionTerminal"/>
   </widget>
   <widget class="QMenu" name="menuFunciones">
    <property name="title">
     <string>Funciones</string>
    </property>
    <addaction name="actionEntre_Matrices"/>
    <addaction name="actionEntre_Escalar"/>
    <addaction name="separator"/>
    <addaction name="actionTrigonometria"/>
    <addaction name="separator"/>
    <addaction name="actionReaolver"/>
    <addaction name="separator"/>
    <addaction name="actionImformacion"/>
    <addaction name="separator"/>
    <addaction name="actionEstadistica"/>
    <addaction name="separator"/>
    <addaction name="actionInterpolar"/>
    <addaction name="actionExtrapolar"/>
    <addaction name="separator"/>
    <addaction name="actionGaficar"/>
    <addaction name="separator"/>
    <addaction name="actionExtras"/>
    <addaction name="separator"/>
    <addaction name="actionLaTex"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMatrices">
    <property name="title">
     <string>Matrices</string>
    </property>
    <addaction name="actionCerrar"/>
    <addaction name="actionRenombrar"/>
    <addaction name="actionCopiar"/>
    <addaction name="actionPegar"/>
   </widget>
   <addaction name="menuArchivos"/>
   <addaction name="menuVer"/>
   <addaction name="menuFunciones"/>
   <addaction name="menuSobre"/>
   <addaction name="menuMatrices"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="ventanaArbol">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="arbol">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="autoExpandDelay">
        <number>1</number>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="terminal">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTextEdit" name="textoTerminal">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrir"/>
   <addaction name="actionAbri_Archivo"/>
   <addaction name="actionNuevo_Proyecto"/>
   <addaction name="actionNuevo_Archivo"/>
   <addaction name="separator"/>
   <addaction name="actionGuardar"/>
   <addaction name="separator"/>
   <addaction name="actionArbol"/>
   <addaction name="separator"/>
   <addaction name="actionReaolver"/>
   <addaction name="actionEstadistica"/>
   <addaction name="actionGaficar"/>
   <addaction name="separator"/>
   <addaction name="actionEntre_Matrices"/>
   <addaction name="actionEntre_Escalar"/>
   <addaction name="separator"/>
   <addaction name="actionTrigonometria"/>
   <addaction name="separator"/>
   <addaction name="actionImformacion"/>
   <addaction name="separator"/>
   <addaction name="actionInterpolar"/>
   <addaction name="actionExtrapolar"/>
   <addaction name="separator"/>
   <addaction name="actionExtras"/>
   <addaction name="separator"/>
   <addaction name="actionLaTex"/>
  </widget>
  <action name="actionNuevo_Proyecto">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/agregar-carpeta.png</normaloff>:/new/prefix1/iconos/agregar-carpeta.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo Proyecto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNuevo_Archivo">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/nuevo.png</normaloff>:/new/prefix1/iconos/nuevo.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo Archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/guardar-el-archivo.png</normaloff>:/new/prefix1/iconos/guardar-el-archivo.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionGuardar_como">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/guardar-el-archivo.png</normaloff>:/new/prefix1/iconos/guardar-el-archivo.png</iconset>
   </property>
   <property name="text">
    <string>Guardar como ...</string>
   </property>
  </action>
  <action name="actionBarra_de_Herramientas">
   <property name="text">
    <string>Barra de Herramientas</string>
   </property>
  </action>
  <action name="actionArbol">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/barra-lateral.png</normaloff>:/new/prefix1/iconos/barra-lateral.png</iconset>
   </property>
   <property name="text">
    <string>Arbol</string>
   </property>
  </action>
  <action name="actionEntre_Matrices">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/matriz3.png</normaloff>:/new/prefix1/iconos/matriz3.png</iconset>
   </property>
   <property name="text">
    <string>Entre Matrices</string>
   </property>
  </action>
  <action name="actionEntre_Escalar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/x.png</normaloff>:/new/prefix1/iconos/x.png</iconset>
   </property>
   <property name="text">
    <string>Entre Escalar</string>
   </property>
  </action>
  <action name="actionTrigonometria">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/trigonometria.png</normaloff>:/new/prefix1/iconos/trigonometria.png</iconset>
   </property>
   <property name="text">
    <string>Trigonometria</string>
   </property>
  </action>
  <action name="actionReaolver">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/resolver.png</normaloff>:/new/prefix1/iconos/resolver.png</iconset>
   </property>
   <property name="text">
    <string>Resolver</string>
   </property>
   <property name="toolTip">
    <string>Resolver</string>
   </property>
  </action>
  <action name="actionImformacion">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/informacion.png</normaloff>:/new/prefix1/iconos/informacion.png</iconset>
   </property>
   <property name="text">
    <string>Imformacion</string>
   </property>
  </action>
  <action name="actionEstadistica">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/probabilidad.png</normaloff>:/new/prefix1/iconos/probabilidad.png</iconset>
   </property>
   <property name="text">
    <string>Estadistica</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/abriProyecto.png</normaloff>:/new/prefix1/iconos/abriProyecto.png</iconset>
   </property>
   <property name="text">
    <string>Abrir Proyecto</string>
   </property>
  </action>
  <action name="actionAbri_Archivo">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/abrirArchivo.png</normaloff>:/new/prefix1/iconos/abrirArchivo.png</iconset>
   </property>
   <property name="text">
    <string>Abri Archivo</string>
   </property>
  </action>
  <action name="actionInterpolar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/interpolar.png</normaloff>:/new/prefix1/iconos/interpolar.png</iconset>
   </property>
   <property name="text">
    <string>Interpolar</string>
   </property>
  </action>
  <action name="actionExtrapolar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/extrapolar.png</normaloff>:/new/prefix1/iconos/extrapolar.png</iconset>
   </property>
   <property name="text">
    <string>Extrapolar</string>
   </property>
  </action>
  <action name="actionGaficar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/graficar.png</normaloff>:/new/prefix1/iconos/graficar.png</iconset>
   </property>
   <property name="text">
    <string>Graficar</string>
   </property>
  </action>
  <action name="actionExtras">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/extra.png</normaloff>:/new/prefix1/iconos/extra.png</iconset>
   </property>
   <property name="text">
    <string>Extras</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/cerrarMatriz.png</normaloff>:/new/prefix1/iconos/cerrarMatriz.png</iconset>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
  <action name="actionRenombrar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/rename.png</normaloff>:/new/prefix1/iconos/rename.png</iconset>
   </property>
   <property name="text">
    <string>Renombrar</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/cortar.png</normaloff>:/new/prefix1/iconos/cortar.png</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/pegar.png</normaloff>:/new/prefix1/iconos/pegar.png</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
  </action>
  <action name="actionLaTex">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/new/prefix1/iconos/OIP.jpg</normaloff>:/new/prefix1/iconos/OIP.jpg</iconset>
   </property>
   <property name="text">
    <string>LaTex</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
